CustomApps/lyrics-plus/ProviderNetease.js

```diff
@@ -19,7 +19,16 @@ const ProviderNetease = (() => {
 		// normalized expected album name
 		const neAlbumName = Utils.normalize(info.album);
 		const expectedAlbumName = Utils.containsHanCharacter(neAlbumName) ? await Utils.toSimplifiedChinese(neAlbumName) : neAlbumName;
-		const itemId = items.findIndex(val => Utils.normalize(val.album.name) === expectedAlbumName || Math.abs(info.duration - val.duration) < 1000);
+		let itemId = items.findIndex(val => Utils.normalize(val.album.name) === expectedAlbumName);
+
+        if (itemId === -1) {
+            itemId = items.findIndex(val => Math.abs(info.duration - val.duration) < 3000);
+        }
+
+        if (itemId === -1) {
+            itemId = items.findIndex(val => val.name === cleanTitle);
+        }
+
```

Comment:
```suggestion
		const itemId = items.findIndex(val => 
			Utils.normalize(val.album.name) === expectedAlbumName ||
    		Math.abs(info.duration - val.duration) < 3000 ||
    		val.name === cleanTitle
		);

```

Reply:
Thank you for your reply, but there may be an album mismatch, and then there may be a market match for that data, but it's not actually the original lyrics, so it's more accurate to iterate over it

Reply:
What you did, with these two ifs, can be shortened into the change I proposed. There's no difference - just better looking code

Reply:
Sure, the translation of the provided code snippet would be:
```javascript
(Utils.normalize(val.album.name) === expectedAlbumName ||
    Math.abs(info.duration - val.duration) < 3000 ) && val.name === cleanTitle

```

Reply:
Then just do how you got it working, however do it inside the `items.findIndex` and not separate if statements

Reply:
Thanks for your help